### PR TITLE
assignment complete

### PR DESCRIPTION
**Pull request by laconiccrafts**

### CODE REVIEW

File: spec/warmups_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+describe Warmups do
+
+  describe "#stock_picker" do
+    it "returns the buy and sell days of most profit" do
+      result = Warmups.stock_picker([44, 30, 24, 32, 35, 30, 40, 38, 15])
+      expect(result).to eq({buy: 2, sell: 6, profit: 16})
+    end
+  end
+
+  describe "#anagrams" do
```
Comment (remis1889) [inline spec/warmups_spec.rb:12]: test for cases when the result is `[]`, when argument is not a `String` etc

File: lib/warmups.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+module Warmups
+
+def self.stock_picker(stocks)
+  result = {buy: nil, sell: nil, profit: nil}
```
Comment (remis1889) [inline lib/warmups.rb:4]: raise an error if the argument is not an `Array` and if the length of stocks array is less than 2

File: spec/warmups_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+require "warmups"
+
+describe Warmups do
+
+  describe "#stock_picker" do
+    it "returns the buy and sell days of most profit" do
```
Comment (remis1889) [inline spec/warmups_spec.rb:6]: test edge cases like `ArgumentError`
